pulsar-broker/src/test/java/org/apache/pulsar/client/api/AuthenticatedProducerConsumerTest.java

```diff
@@ -80,6 +80,7 @@ protected void setup() throws Exception {
         conf.setTlsCertificateFilePath(TLS_SERVER_CERT_FILE_PATH);
         conf.setTlsKeyFilePath(TLS_SERVER_KEY_FILE_PATH);
         conf.setTlsAllowInsecureConnection(true);
+        conf.setTopicLevelPoliciesEnabled(false);
```

Comment:
Because TLS is not configured for the internal Pulsar client, we get an exception when we delete the topic policy.

I'm wondering if we enable `topicPolicies` and TLS but don't provide any TLS configuration for the broker's internal clients.
Do we have to check the broker`s internal client configuration when the broker starts?

Reply:
@mattisonchao Please open a new issue for this one.